packages/mdc-icon-button/index.ts

```diff
@@ -22,34 +22,27 @@
  */
 
 import MDCComponent from '@material/base/component';
-import MDCIconButtonToggleFoundation from './foundation';
 import {MDCRipple} from '@material/ripple/index';
+import MDCIconButtonToggleFoundation from './foundation';
 
-/**
- * @extends {MDCComponent<!MDCIconButtonToggleFoundation>}
- */
-class MDCIconButtonToggle extends MDCComponent {
-  static attachTo(root) {
+class MDCIconButtonToggle extends MDCComponent<MDCIconButtonToggleFoundation> {
+  static attachTo(root: Element) {
     return new MDCIconButtonToggle(root);
   }
 
-  constructor(...args) {
-    super(...args);
+  private ripple_: MDCRipple;
+  private handleClick_: EventListenerOrEventListenerObject;
```

Comment:
Did the @material/Dom import not work?

Reply:
if you add the line below you can replace `EventListenerOrEventListenerObject ` --> `private handleClick_: SpecificEventListener<'click'>;`


```
root_!: HTMLElement;
```